src/content/app/genome-browser/services/browserStorageService.ts

```diff
@@ -15,26 +15,31 @@
  */
 
 import storageService, {
-  StorageServiceInterface
+  type StorageServiceInterface
 } from 'src/services/storage-service';
-import {
+import type {
   BrowserTrackStates,
   GenomeTrackStates
 } from '../components/track-panel/trackPanelConfig';
 
-import { ChrLocation } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
-import {
+import type { ChrLocation } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSlice';
+import type {
   TrackPanelState,
   TrackPanelStateForGenome
 } from 'src/content/app/genome-browser/state/track-panel/trackPanelSlice';
+import type {
+  BrowserBookmarksState,
+  BrowserBookmarksStateForGenome
+} from '../state/browser-bookmarks/browserBookmarksSlice';
 
 export enum StorageKeys {
   ACTIVE_GENOME_ID = 'browser.activeGenomeId',
   ACTIVE_ENS_OBJECT_ID = 'browser.activeFocusObjectId',
   CHR_LOCATION = 'browser.chrLocation',
   DEFAULT_CHR_LOCATION = 'browser.defaultChrLocation',
   TRACK_STATES = 'browser.trackStates',
-  TRACK_PANELS = 'browser.trackPanels'
+  TRACK_PANELS = 'browser.trackPanels',
+  BOOKMARKS = 'browser.bookmarks'
```

Comment:
Now that bookmarks are in the bookmarks service, are we still using this key?

Also, would there be a benefit in renaming the first part of these keys to genomeBrowser?

Reply:
This is related to my comment below. The bookmarks key can be deleted if we get rid of the code related to it here. 

And about renaming from `browser` to `genomeBrowser`, is this because the keys are related to the genome browser itself (wasm) rather than the app (named `Browser` at the moment)?

Also should we create a new `trackPanelStorageService` for the `TrackPanel` code (like how bookmarks has one for itself)?

Reply:
We decided do this later in a separate PR